test/Amadevus.RecordGenerator.Test/RecordFormatterTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Immutable;
+using Xunit;
+
+namespace Amadevus.RecordGenerator.Test
+{
+    public class RecordFormatterTests : RecordTestsBase
+    {
+        [Fact]
+        public void With_SimpleProperty()
+        {
+            var item = CreateItem();
+            var shape = new
+            {
+                Id = ItemId,
+                Name = ItemName
+            };
+            Assert.Equal(shape.ToString(), item.ToString());
+        }
+
+        [Fact]
+        public void With_CollectionProperty()
+        {
+            var item = CreateItem();
+            var container = CreateContainerEmpty();
+            var items = new[] { item }.ToImmutableArray();
+            container.WithItems(items);
+            var shape = new
+            {
+                Id = ContainerId,
+                Name = ContainerName,
+                Items = items.ToString(),
+            };
+            Assert.Equal(shape.ToString(), container.ToString());
+        }
+    }
```

Comment:
Could you add a test that uses GenericRecord with Item type parameter to validate nested record case?

Reply:
Done in 4245fa13a721d5b7634abde68c60ca1cfb44a9c3.